datumaro/util/meta_file_util.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from collections import OrderedDict
+import json
+import os.path as osp
+
+from datumaro.util import find
+
+DATASET_META_FILE = 'dataset_meta.json'
+
+def parse_meta_file(path):
+    meta_file = path
+    if (osp.isdir(path)):
+        meta_file = osp.join(path, DATASET_META_FILE)
+
+    with open(meta_file) as f:
+        dataset_meta = json.load(f)
+
+    label_map = OrderedDict()
+    colors = dataset_meta.get('segmentation_colors', [])
+    parts = dataset_meta.get('parts', {})
+    actions = dataset_meta.get('actions', {})
```

Comment:
Think if we can support these two fields another way, without polluting the general function scope and interface. Maybe, it should be redefined or extended for VOC specifically.

Reply:
Added separate functions for this format.